app/src/main/java/org/simple/clinic/introvideoscreen/IntroVideoEvent.kt

```diff
@@ -1,6 +1,8 @@
 package org.simple.clinic.introvideoscreen
 
-sealed class IntroVideoEvent
+import org.simple.clinic.widgets.UiEvent
+
+sealed class IntroVideoEvent : UiEvent
```

Comment:
Can you add analytics event names for the `VideoClicked` and `SkipClicked` events?